src/client/features/enrichment/token-input.js

```diff
@@ -67,9 +70,44 @@ class TokenInput {
     });
   }
 
+
+ render() {
+
+  return (
+    //titleContainer: [
+      [h('h4', [
+        h('span', 'Pathway Enrichment   '),]),
+      h('img', {
+        src: '/img/humanIcon.png'
+        }),
+      h('div.gene-input-container', [
```

Comment:
Everything above the `gene-input-container` should go in parent. Everything below can stay.